java/HealthVault Mobile/HVSample/src/main/java/com/microsoft/hsg/android/hvsample/AddMedicationActivity.java

```diff
@@ -27,18 +28,109 @@
 import android.widget.ArrayAdapter;
 import android.widget.Button;
 import android.widget.EditText;
+import android.widget.LinearLayout;
 import android.widget.ListView;
 import android.widget.Spinner;
 import android.widget.TextView;
 import android.widget.Toast;
 import android.widget.AdapterView.OnItemSelectedListener;
 
 public class AddMedicationActivity extends Activity {
+	private HealthVaultApp mService;
+	private HealthVaultClient mHVClient;
+	private Record mCurrentRecord;
+	private int mIndex = 0;
+
 	@Override
 	protected void onCreate(@Nullable Bundle savedInstanceState) {
 		super.onCreate(savedInstanceState);
 		setContentView(R.layout.add_medication);
+		Button editButton = (Button) findViewById(R.id.editdetails_button);
+		mService = HealthVaultApp.getInstance();
+		mHVClient = new HealthVaultClient();
+		mCurrentRecord = HealthVaultApp.getInstance().getCurrentRecord();
+
+		Intent mIntent = getIntent();
+		mIndex = mIntent.getIntExtra(Constants.IndexParameter, 0);
+
+		editButton.setEnabled(false);
```

Comment:
Why do we have this button if it is disabled? I don't see it being set to enabled anywhere. If it is always going to be disabled, you can do that in xml and avoid the view lookup here, but currently I don't see the point of this button.